src/common-middleware.js

```diff
@@ -0,0 +1,84 @@
+// Setup sharify
+// TODO: Export a function instead of loading on import.
+import "./lib/setup_sharify"
+
+import addRequestId from "express-request-id"
+import compression from "compression"
+import cookieParser from "cookie-parser"
+import express from "express"
+import favicon from "serve-favicon"
+import glob from "glob"
+import helmet from "helmet"
+import path from "path"
+import sharify from "sharify"
+import siteAssociation from "artsy-eigen-web-association"
+import timeout from "connect-timeout"
+import ensureSSL from "./lib/middleware/ensure_ssl"
+import hstsMiddleware from "./lib/middleware/hsts"
+import ipFilterMiddleware from "./lib/middleware/ipFilter"
+import sessionMiddleware from "./lib/middleware/sessionMiddleware"
+import config from "./config"
+
+const { APP_TIMEOUT, IP_DENYLIST, NODE_ENV } = config
+
+function securityMiddleware(app) {
+  // Denied IPs
+  if (IP_DENYLIST && IP_DENYLIST.length > 0) {
+    app.use(ipFilter(IP_DENYLIST))
+  }
+
+  // Timeout middleware
+  if (process.env.NODE_ENV === "production") {
+    app.use(timeout(APP_TIMEOUT || "29s"))
+  }
+
+  // Make sure we're using SSL and prevent clickjacking
+  app.use(ensureSSL)
+
+  // Use HSTS to prevent downgrade attacks.
+  app.use(hstsMiddleware)
+
+  // Prevent frame nesting in development and production.
+  if (!NODE_ENV === "test") {
+    app.use(helmet.frameguard())
+  }
+
+  // Inject UUID for each request into the X-Request-Id header
+  app.use(addRequestId())
+
+  // Cookie parser
+  app.use(cookieParser())
+
+  // Session cookie
+  app.use(sessionMiddleware())
+}
+
+function staticAssetMiddlewares(app) {
+  // Mount static assets from root public folder
+  app.use(express.static("public"))
+
+  // Mount static assets from sub-app /app `public` folders
+  glob
+    .sync(`${__dirname}/{public,{desktop,mobile}/**/public}`)
+    .forEach(folder => {
+      app.use(express.static(folder))
+    })
+
+  app.use(favicon(path.resolve(__dirname, "mobile/public/images/favicon.ico")))
+  app.use("/(.well-known/)?apple-app-site-association", siteAssociation)
+}
+
+export default function commonMiddlewareSetup(app) {
```

Comment:
New files should be avoiding named exports per long standing Artsy convention

Reply:
As per the previous conversation, there was a workstream to clean up these exports immediately following this change. I will continue to hold this PR until that work has been completed per your request.

It would be nice if you would provide better clarity and history of this convention. There several changes in recent history that have violated this convention without need. My suggestion is that we codefiy our conventions, this is exactly what linters do and the PR would provide history for when and why a rule was changed.

Reply:
We've already gone over this [numerous times  in different contexts](https://artsy.slack.com/archives/CA8SANW3W/p1604635470483500). I can explain it again: Every codebase at artsy avoids them for new code. We've refactored them out and socialized the practice. There's not a lint rule in force because there's an exception to the rule -- we can't bundle split without them. In every instance we do use them there's a comment associated. 

What i'm asking for here is not complicated or unclear. Its a convention. 

To repeat yet again: if you feel strongly about default exports we have an RFC process you can tap into to argue your point.

Reply:
> I  will continue to hold this PR

You don't need to do that, it takes less than 30 seconds to change 

```tsx
export default function commonMiddlewareSetup
```
to 
```tsx
export function commonMiddlewareSetup
```

in each new spot and then adjust the corresponding import.